hub/load-test/Chart.yaml

```diff
@@ -19,4 +19,5 @@ maintainers:
   url: https://researcher.watson.ibm.com/researcher/view.php?person=us-spartha
 icon: https://github.com/iter8-tools/iter8/raw/master/mkdocs/docs/images/favicon.png
 appVersion: "0.8"
+deprecated: true
 
```

Comment:
We've already submitted a Knative PR (which got merged) with the load-test chart. So, simply dropping it (preview or otherwise), doesn't feel right. What happens to users who, lets say, already built a CI/CD pipeline with the load-test chart? 
***
The above concern is a bit theoretical at this point... and I have already changed the link/docs to point to `load-test-http`... However, I think we want to get the deprecation process right. The `load-test` deprecation gives us an opportunity to do so.